Makefile

```diff
@@ -1,6 +1,6 @@
 include golang.mk
 .DEFAULT_GOAL := test # override default goal set in library makefile
-.PHONY: test build
+.PHONY: test build hardcoded.go
```

Comment:
Making the `hardcoded.go` target PHONY means that it will be regenerated even if none of its dependencies have changed. Right now the main dependencies are the contents of the hardcoded directory (`hardcoded/*` below), which seems sufficient to keep the target non-PHONY? Were you seeing `make` not update this file when changed were made to this directory?